### ISSUE TITLE
"Ok" button on blocked mods workaround needs improvement

### ISSUE THREAD

**Issue by Poopooracoocoo** [PrismLauncher/PrismLauncher#613]
### Role

I download mods from CurseForge that have opted out of third-party distribution

### Issue

When you download a CurseForge modpack with opted-out mods such as Sevtech Ages or Fabulously Optimized, you'll see Prism's workaround to download them. 

"Ok" button is not clear. It's too easy to click on it right away and assume that Prism will deal with it for me. 

![image](https://user-images.githubusercontent.com/51363864/207297169-3c31d899-2f26-4722-afad-6ad6641a0931.png)

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### Suggestion

Consider greying it out until all mods are found or changing it to "Dismiss", "Continue" or "Skip" until all mods are found.

When all mods are found, it could be called "Done".

Consider moving the "Add download folder" to another line so that it's more visually related to the watched folders area.

Or some kind of other change to the UI. Idk.

Here's a mockup. The "Skip" button would be called "Done" when all mods are found. 

![image](https://user-images.githubusercontent.com/51363864/232737578-5b801cea-e0ab-44cf-8787-3f0cb76b6cca.png)

**Comment by clarfonthey** [comment]
Yeah, want to second this. It's especially egregious because the window is sized as if there's only one mod for me, and I can hit OK because it looks like I've got all the mods, but I'm actually missing some.

This definitely could be improved.
